examples/around.rs

```diff
@@ -58,10 +58,11 @@ fn main() {
     let silent = Iron::new(Logger::new(LoggerMode::Silent).around(Box::new(hello_world)));
     let large = Iron::new(Logger::new(LoggerMode::Large).around(Box::new(hello_world)));
 
+    println!("Starting servers on 2000, 3000, and 4000");
+    
     tiny.listen("localhost:2000").unwrap();
```

Comment:
I think this example is actually incorrect now - we'll block here and never set up `silent` or `large`.

Can you change this to `let _tiny_listening = tiny..` for each of these so that creating the server doesn't block, then bring back the println at the bottom?